--- a/webview/assets/app-initial~app-main~onboarding-page~hotkey-window-thread-page~quick-chat-window-page~chatg~k0ede4gb-C17KDkOa.js
+++ b/webview/assets/app-initial~app-main~onboarding-page~hotkey-window-thread-page~quick-chat-window-page~chatg~k0ede4gb-C17KDkOa.js
@@ -46984,6 +46984,9 @@
       (n[2] = r))
     : (r = n[2]);
   let i = r;
+  window.__ELECTRON_SHIM__.closeSidebar = () => {
+    kl(e, !1, { animate: t });
+  };
   Op(`toggleSidebar`, i);
   let a;
   (n[3] !== e || n[4] !== t
